Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -0,0 +1,5783 @@
+    SEVERITY = 'The severity should be High/Medium/Low/Info'
+    IGNORE_X_FORWARDED_FOR = 'ignore_x_forwarded_for should be enable/disable'
+    V1_NOT_SUPPORTED = 'Command not supported in version 1.'
+    TYPE = 'The type should be "Allow Only Ip"/"Black IP"/"Trust IP"'
+    IP = 'is not a valid IPv4/IPv6 address.'
+    ALLOW_IP_V1 = 'Allow only ip not supported by version 1.'
+    IS_DEFAULT = 'is_default should be yes/no'
+    INHERIT_WEB_PROTECTION_PROFILE = 'inherit_web_protection_profile should be enable/disable'
+    STATUS = 'status should be enable/disable'
+    PROTOCOL = 'It must to insert at least one HTTP or HTTPS service.'
+    DEPLOYMENT_MODE = 'deployment_mode should be "HTTP Content Routing"/"Single Server/Server Balance"'
+    SERVER_POOL = 'Server pool is requierd argument while deployment_mode is "Single Server/Server Balance".'
+    SCRIPTING = 'scripting should be enable/disable'
+    SCRIPTING_LIST = 'At Least one scripting is required.'
+    CERTIFICATE_TYPE = 'certificate_type should be "Local"/"Multi Certificate"/"Letsencrypt"'
+    REQUEST_URL = 'Request URL must start with  / .'
+    REQUEST_URL_INSERT = 'Please insert request_url.'
+    REQUEST_TYPE = 'request_type should be "Simple String"/"Regular Expression"'
+    DOMAIN_TYPE = 'domain_type should be "Simple String"/"Regular Expression"'
+    NAME_TYPE = 'name_type should be "Simple String"/"Regular Expression"'
+    HEADER_NAME_TYPE = 'header_name_type should be "Simple String"/"Regular Expression"'
+    HEADER_VALUE_TYPE = 'header_value_type should be "Simple String"/"Regular Expression"'
+    DOMAIN_INSERT = 'Please insert domain.'
+    VALUE_INSERT = 'Please insert value.'
+    COUNTRIES = 'Please insert counries from the list.'
+    NAME_INSERT = 'Please insert name.'
+    DEPLOYMENT_MODE_INSERT = 'Please insert deployment mode.'
+    VIRTUAL_SERVER = 'Please insert virtual server.'
+    CLIENT_REAL_IP = 'client_real_ip should be enable/disable'
+    MACH_ONCE = 'mach_once should be enable/disable'
+    MONITOR_MODE = 'monitor_mode should be enable/disable'
+    REDIRECT_2_HTTPS = 'redirect_to_https should be enable/disable'
+    RETRY_ON = 'retry_on should be enable/disable'
+    RETRY_ON_HTTP_LAYER = 'retry_on_http_layer should be enable/disable'
+    RETRY_ON_CONNECT_FAILURE = 'retry_on_connect_failure should be enable/disable'
+    SYN_COOKIE = 'syn_cookie should be enable/disable'
+    URL_CASE_SENSITIVITY = 'url_case_sensitivity should be enable/disable'
+    HALF_OPEN_THRESH = 'half_open_thresh should be a number in range of 10-10,000.'
+    RETRY_TIMES_ON_CONNECT = 'retry_times_on_connect_failure should be a number in range of 1-5.'
+    RETRY_TIMES_ON_HTTP = 'retry_times_on_http_layer should be a number in range of 1-5.'
+    RETRY_ON_HTTP_RESPONSE_CODES = 'Please insert codes from the list.'
+
+
+class Parser:
+
+    @abstractmethod
+    def protected_hostname_group(self, protected_hostname_group: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
All these commands under the Parser class. when I want to user them I write parser.protected_blablalba...

Reply:
We use OOP to describe in better abstraction relations between different software components. In general those software components can be classes, methods .etc. What is the relation between the parser and protected_blablalba?
In my view, the parser is an entity which have specific properties and capabilities. one of her capabilities is protected_blablalba. When I try to describe it in a sentence it looks like something is missing.  The **parser** did **protected_blablalba**? I think it will be better if we will have: The **parser** parsed **protected_blablalba*. Think about it. we can talk about it furtherly tomorrow.